src/main/java/uk/gov/hmcts/cmc/claimstore/courtfinder/models/Address.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.hmcts.cmc.claimstore.courtfinder.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
Are the no-args constructors really necessary?

Reply:
We usually create a builder for the domain objects, which implicitly creates the constructors